website/data/docs-navigation.js

```diff
@@ -49,8 +49,9 @@ export default [
         content: ['scheduling', 'preemption'],
       },
       'consensus',
+      'filesystem',
```

Comment:
I'm not 100% certain on the ordering here. It looks like this might be alphabetical, but with Architecture and sections first?